JS.md

```diff
@@ -262,11 +262,6 @@ Set `moduleRecord.[[Namespace]]` to `moduleNamespace`.
 Let `instanceObject` be a new `WebAssembly.Instance` object setting
 `[[Instance]]` to `instance` and `exports` to `moduleNamespace`.
 
-If any of the elements of an [Elements section](Modules.md#elements-section)
-refers to an imported function which is not an
-[Exported Function Exotic Object](#exported-function-exotic-objects), throw a
-[`TypeError`](https://tc39.github.io/ecma262/#sec-native-error-types-used-in-this-standard-typeerror).
```

Comment:
There is a similar check above (GH won't give me a + to comment on it directly) preventing host functions in elems section that should be removed as well.